python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -13,6 +13,9 @@
 
 logger = getLogger()
 
+_reg_id_str = '_registry_id'
+_id_str = 'id'
+
```

Comment:
I'm wanting to remove registry_id and try replacing it with 'id' directly. I don't know how good/bad that would be so I'm wanting to play around with this parameter on a private branch.
Doing this as an exercise showed up at least one bug though so the result is getting committed even if it doesn't effect much